network-api/networkapi/wagtailpages/templates/buyersguide/product_page.html

```diff
@@ -213,18 +213,18 @@ <h3 class="tw-mb-4 tw-font-zilla tw-text-2xl tw-leading-7">{% trans "What data d
                   </section>
 
                   {% trans "How does the company use this data?" as how_does_it_use_data_collected %}
-                  {% include "fragments/product_criterion.html"  with  label=how_does_it_use_data_collected  value=product.how_does_it_use_data_collected  ding=product.data_collection_policy_is_bad %}
+                  {% include "fragments/product_criterion.html"  with  label=how_does_it_use_data_collected  value=product.how_does_it_use_data_collected  ding=product.data_collection_policy_is_bad rich_text_value=True %}
```

Comment:
Per request, I created a new ticket to update this template to have it sense whether or not it is using a rich text value, instead of having to set it through a boolean each time. 

https://github.com/mozilla/foundation.mozilla.org/issues/7755